sdcm/cluster.py

```diff
@@ -441,6 +442,12 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
         self._kernel_version = None
         self._cassandra_stress_version = None
 
+        # Next 3 parameters are used and allowed to be changed by unit tests in unit_tests/test_decode_backtrace.py.
+        # Want to change it to reduce the test time
+        self.backtrace_lookup_period: int = 15
+        self.get_core_pids_func_retrying: int = 10
+        self.get_core_pids_func_sleep_timeout: int = 20
+
```

Comment:
I meant these variables too. Please, move them to class level.

Reply:
done